uv.lock

```diff
@@ -1,5 +1,5 @@
 version = 1
-revision = 3
+revision = 2
```

Comment:
You probably don't have the lastest version of uv. Could you upgrade it and rebuild the lock?